src/commands/synthetics/README.md

```diff
@@ -35,115 +33,66 @@ yarn add --dev @datadog/datadog-ci
+Using a global configuration file is one of the ways to configure datadog-ci. To do so, create a JSON configuration file on your system. Specify the path to the file using the `--config` flag or configure it through the `DATADOG_SYNTHETICS_CONFIG_PATH` environment variable [when launching your tests](#run-tests-command) or [uploading a new application](#upload-application-command). If you don't specify a file path, Datadog looks for a file with the default filename of `datadog-ci.json`.
 
-`mobileApplicationVersionFilePath`
-: Override the application version for all Synthetic mobile application tests.
-
-`batchTimeout`
-: **Type**: integer<br>
-The duration (in milliseconds) after which `datadog-ci` stops waiting for test results. The default is 30 minutes. At the CI level, test results completed after this duration are considered failed.
-
-`proxy`
-: The proxy to be used for outgoing connections to Datadog. `host` and `port` keys are mandatory arguments, the `protocol` key defaults to `http`. Supported values for the `protocol` key are `http`, `https`, `socks`, `socks4`, `socks4a`, `socks5`, `socks5h`, `pac+data`, `pac+file`, `pac+ftp`, `pac+http`, and `pac+https`. The library used to configure the proxy is the [proxy-agent][2] library.
-
-`selectiveRerun`
-: A boolean flag to only run the tests which failed in the previous test batches. Use the `--no-selectiveRerun` CLI flag to force a full run if your configuration enables it by default.
-
-`subdomain`
-: The name of the custom subdomain set to access your Datadog application. If the URL used to access Datadog is `myorg.datadoghq.com`, the `subdomain` value needs to be set to `myorg`.
-
-`testSearchQuery`
-: Pass a query to select which Synthetic tests to run. If you are running tests in the CLI, use the `-s` flag.
-
-`tunnel`
-: Use [Local and Staging Environments](#use-local-and-staging-environments) to execute your test batch.
-
-**Note**: The `pollingTimeout` option and `--pollingTimeout` CLI parameter are deprecated in favor of `batchTimeout` and `--batchTimeout`, respectively.
-
-#### Use a proxy
-
-It is possible to configure a proxy to be used for outgoing connections to Datadog using the `proxy` key of the global configuration file.
-
-As the [`proxy-agent` library][2] is used to configure the proxy, the supported protocols include `http`, `https`, `socks`, `socks4`, `socks4a`, `socks5`, `socks5h`, `pac+data`, `pac+file`, `pac+ftp`, `pac+http`, and `pac+https`. The `proxy` key of the global configuration file is passed to a new `proxy-agent` instance, which means the same configuration for the library is supported.
+See each command's list of configurations below for the list of advanced options in the global configuration file relevant to each [run-tests command](#list-of-configurations) and [upload-application command](#list-of-configurations-1). For an example configuration file, see this [`example-global-config.json` file][9].
```

Comment:
So originally we were using the `.github/workflows/e2e/global.config.json` and i was supposed to change that file to just include more info in [this PR](https://github.com/DataDog/datadog-ci/pull/1368), but then i realised that that file is actually used for something and is not just an example. So i opted to create a separate file just as an example instead.

As for if we should move it next to the README, considering that all of the other "config" files are in the `.github/workflows/e2e/` folder i think it makes more sense to leave the file in that folder. Additionally in terms of keeping this information close to the README the example just below is pretty much the same as this example file, so that should be good enough i think 🤔

Reply:
@Drarig29 managed to convince me to move it :D 
Moved it to a examples folder in the synthetics folder 👍